### DISCUSSION TITLE
Full-fledged multi-track editor like Cool Edit Pro

### DISCUSSION THREAD

**Discussion by native-engine** [audacity/audacity#1757] [General]
Aren't you planning to make Audacity a full-fledged multi-track editor like Cool Edit Pro?
It was a great sound editor until Adobe acquired and messed up its interface.

**Comment by LWinterberg** [comment]
The plan generally is to make Audacity easier to use and maintain while retaining its current feature set, while also making it more capable as a full-on DAW. We won't be looking to replicate the way Cool Edit (or Audition, Ableton, Pro Tools, ...) are doing things though, because Audacity has a very different userbase to all of them, namely casual users. So when we expand our multi-track capabilities, we'll need to make sure these concepts are understood by these casual users, as opposed to just following potentially obscure conventions which have existed in the industry for decades.
